server/src/test/java/io/crate/execution/engine/aggregation/impl/StdDevSampAggregationTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.execution.engine.aggregation.impl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.Test;
+
+import io.crate.exceptions.UnsupportedFunctionException;
+import io.crate.expression.symbol.Literal;
+import io.crate.metadata.FunctionImplementation;
+import io.crate.metadata.FunctionType;
+import io.crate.metadata.Scalar;
+import io.crate.metadata.SearchPath;
+import io.crate.metadata.functions.Signature;
+import io.crate.operation.aggregation.AggregationTestCase;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+import io.crate.types.NumericType;
+
+public class StdDevSampAggregationTest extends AggregationTestCase {
+
+    private Object executeAggregation(DataType<?> argumentType, Object[][] data) throws Exception {
+        return executeAggregation(
+                Signature.builder("stddev_samp", FunctionType.AGGREGATE)
+                        .argumentTypes(argumentType.getTypeSignature())
+                        .returnType(DataTypes.DOUBLE.getTypeSignature())
```

Comment:
For numeric types, the return type should be numeric, I wonder a bit that this works with a slightly wrong signature.